x-pack/legacy/plugins/ml/public/explorer/explorer.js

```diff
@@ -789,7 +789,8 @@ export const Explorer = injectI18n(injectObservablesAsProps(
       if (stateUpdate.influencers !== undefined && !noInfluencersConfigured) {
         for (const influencerName in stateUpdate.influencers) {
           if (stateUpdate.influencers[influencerName][0] && stateUpdate.influencers[influencerName][0].influencerFieldValue) {
-            stateUpdate.filterPlaceHolder = `${influencerName} : ${stateUpdate.influencers[influencerName][0].influencerFieldValue}`;
+            stateUpdate.filterPlaceHolder =
+            `Filter by influencer fields... (E.g. ${influencerName} : ${stateUpdate.influencers[influencerName][0].influencerFieldValue})`;
```

Comment:
I suspect this now needs to be translated? Can you maybe reuse the translation id `xpack.ml.explorer.kueryBar.filterPlaceholder` if it's the same message and was just moved to another place?